src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,91 @@
+package duke.task;
+
+/**
+ * Stores the description of a task and whether it is done.
+ */
+public abstract class Task {
+  private static final String DONE = "X";
```

Comment:
I like how you extracted out the string here! One suggestion is using enum or put all the string values in UI and make it a static variables?